common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1169,6 +1174,11 @@
          */
         public static final String REDIRECT_SSL_PREFIX = "https://";
 
+        /**
+         * Prefix in the redirect for PlayStore.
+         */
+        public static final String PLAY_STORE_INSTALL_PREFIX = "market://details?id=";
```

Comment:
I don't know if we should bank on this URI being this stable.  We could have other query parameters, for sure, that we should probably ignore.  If it was critical, you'd think they'd make it part of the path.

Reply:
its not a part of the path as of now, if they do that, other libraries including ours would have to make the changes.
But as of now, since Auth app being the primary choice to handle MFA, we can be certain to get an URI such as this hinting to download authenticator app.

Reply:
This is more about being humble with respect to query parameters that might appear in that string.  It could change, somebody could decide to add a cache breaker or what-have-you.

As a contrived example:

https://play.google.com/store/apps/details?id=com.zhiliaoapp.musically&foo=bar

This uri works.

We might not want to allow it, but if we do block it, we should say why (we found unknown query paratmeters in the uri).

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1195/commits/fa6aa08d511b6cc2d891b8ec8e9e9a13a542de2b